src/query/useSubmitRpcTx.ts

```diff
@@ -39,6 +45,17 @@ export const useSubmitRpcTx = () => {
         const sentTx = await rpcServer.sendTransaction(transaction);
 
         if (sentTx.status !== "PENDING") {
+          if (sentTx.errorResult) {
+            const errorResult = parse(
+              StellarXdr.decode(
+                "TransactionResult",
+                sentTx.errorResult?.toXDR("base64"),
+              ),
```

Comment:
Should we make sure that `StellarXdr` is available (`useIsXdrInit()`)?

Reply:
thank you! I will use `await StellarXdr.initialize();`